templates/concept.html

```diff
@@ -85,7 +85,9 @@
 			{{#unless items.length }}
 			<p class="topic-card__excess">0 stories</p>
 			{{/unless}}
-			<div class="topic-card__myft-btn topic-card__myft-btn--alerts">{{> n-myft-ui/components/instant-alert/instant-alert conceptId=id name=name extraClasses="topic-card__concept-instant" }}</div>
+			<div class="topic-card__myft-btn topic-card__myft-btn--alerts">
+				{{> n-myft-ui/components/instant-alert/instant-alert conceptId=id name=name }}
```

Comment:
Is the `extraClasses="topic-card__concept-instant"` not needed anymore?

Reply:
I wasn't able to find the class with this name anywhere in our code base, let me know if I missed it somewhere

Reply:
(no trace of it anywhere in our codebase, and w/o it all is the same)

Reply:
y, this very line is only place where it is mentioned https://github.com/search?q=org%3Afinancial-times+topic-card__concept-instant&type=Code